src/daemonizer/posix_fork.cpp

```diff
@@ -28,7 +28,7 @@ namespace {
   }
 }
 
-void fork()
+void fork(std::string const & pidfile)
```

Comment:
I know this is really picky but stylistically (and in most, if not all, of the monerod code) const is before the type.